resources/styles/organisms/site-footer/_index.scss

```diff
@@ -15,32 +15,57 @@
         align-items: start;
     }
 
-    &__links {
-        grid-column: 5 / 11;
-        justify-self: center;
+
+    &__identity {
+        display: grid;
+        grid-column: main;
+        row-gap: config.$thin-h-space;
+
+        @include at(lg) {
+            grid-column: 2 / 5;
+        }
     }
 
     &__brand {
-        grid-column: 2 / 5;
         align-items: center;
-        color: config.$primary;
+        color: config.$yellow-100;
```

Comment:
Let's stick to either a shade of `$primary` (you can pick a lighter one for contrast) or `$secondary` for brand colors.

Reply:
i used $blue-300